crates/release_plz_core/src/command/release_pr.rs

```diff
@@ -274,10 +303,15 @@ fn update_pr_branch(
     Ok(())
 }
 
-fn reset_branch(pr: &GitPr, commits_number: usize, repository: &Repo) -> anyhow::Result<()> {
+fn reset_branch(
+    pr: &GitPr,
+    commits_number: usize,
+    repository: &Repo,
+    branch_prefix: &str,
+) -> anyhow::Result<()> {
     // sanity check to avoid doing bad things on non-release-plz branches
     anyhow::ensure!(
-        pr.branch().starts_with(BRANCH_PREFIX) || pr.branch().starts_with(OLD_BRANCH_PREFIX),
+        pr.branch().starts_with(branch_prefix) || pr.branch().starts_with(OLD_BRANCH_PREFIX),
```

Comment:
let's add here `|| pr.branch().starts_with(BRANCH_PREFIX)`.
Not sure if it's needed but just to be safe.

Reply:
Added the extra check, just to be safe!